apps/metricstream/template.yaml

```diff
@@ -186,9 +238,101 @@ Resources:
         Name: "AWS::Include"
         Parameters:
           Location: !Ref FilterUri
-      # end of processable content for include
+      # End of processable content for include.
+      # The include function here is a macro that
+      # will be replaced with the contents of the file
+      # specified by the FilterUri parameter.
+      # The filterURI parameter is a decided in the
+      # parent stack, where, depending on whether we are
```

Comment:
can you install this AWS Sam app separately without installing the whole stack? you should be able to